client/src/game/gameObjects/moveableGameObject.ts

```diff
@@ -25,36 +25,35 @@ class MoveableGameObject extends GameObject implements IMoveableGameObject {
 				}
 			}
 		}
-
 		super.update(time, delta)
 		// updateData('position')
 	}
 
 	private move(source: ICordinates, destination: ICordinates, delta: number): ICordinates {
```

Comment:
Sorry, this is not related to your PR, but do you guys prefix interfaces with I..? Isn't that too ancient for TS? Also it's coordinates not cordinates 👍 Feel free to resolve. This is just for clarification

Reply:
If I for some conceivable reason need to change ICoordinates from interface to type, it makes even less sense to prefix with I.

Reply:
It depends on the language:
I prefix on interfaces is a C# thing, and normally you would then in the implementers omit the I.
IStrategy (interface) and Strategy (class).

In java you would omit the I on the interface, but add a Impl on the class.
Strategy (interface) and StrategyImpl (class).

In typescript, there is no real "default". But i pefer to omit the I for the following reasons:
1. It is only really C# that prefixes with I
2. I prefer that the users of a type does not know if it is an interface or a class, they should not care. If I at one point have a class and later change it to an interface users should not care.
3. interfaces in typescirpt are different form java/c# interfaces, since typescirpt uses https://en.wikipedia.org/wiki/Structural_type_system.

Reply:
I don't prefix interfaces with I anywhere. AFAIK it's really a Microsoft thing (another ancient practice), and then adopted by .NET devs. Either way, it seems we agree on not using I prefix on interfaces in TS 👍